### PR TITLE
Add Openshift setup and test

### LINKED ISSUE #465: Automate openshift - CC deployment and testing
Similar to what was done with k8s, we need to automate the deployment of Openshift on top of ClearContainers and execute at least mininal testing.
Following steps would be to to add this automation into the CI and increase the testing for this integration.

### PR DESCRIPTION
**Pull request by chavafg**
This commit adds scripts to setup the environment for running OpenShift on top of Clear Containers, initialize the nodes and run a simple Connectivity test.

Fixes #465.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

File: integration/openshift/init.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# limitations under the License.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/openshiftrc"
+
+echo "Disable SELinux"
+echo "There is an issue when runnig CRI-O workloads with SELinux enabled
+For more information, please take a look at:
```
Comment (egernst) [inline integration/openshift/init.sh]: nice, thanks salvador!

File: integration/openshift/openshiftrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This variables will be used for launching the cluster and
```
Comment (egernst) [inline integration/openshift/openshiftrc]: s/This/These

**Comment by chavafg** [comment]
@egernst thanks for your comments. 

I added a verification on the scripts to state that currently these scripts only work on Fedora. 
Also added a README.md with that information and how to execute the scripts.

**Comment by chavafg** [comment]
@iphutch can you take a look at the README created here? thanks

File: integration/openshift/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Openshift integration with Clear Containers
```
Comment (iphutch) [inline integration/openshift/README.md]: Replace Openshift with OpenShift\*
The * is to satisfy the trademark in the first mention.

File: integration/openshift/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+In this section you can find scripts to setup an Openshift enviornment
```
Comment (iphutch) [inline integration/openshift/README.md]: OpenShift should always have a capitalized S.
Could replace "In this section you can find" with "This section contains"

File: integration/openshift/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+on top of Clear Containers. Currently, these scripts only work for **Fedora**.
+
+Please execute the steps below on a Clean Environment.
```
Comment (iphutch) [inline integration/openshift/README.md]: Use lower case for "clean environment" and remove "Please" to start the sentence with an action.

File: integration/openshift/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Setup the environment:
+
+```
+$ ./setup.sh
+```
+
+Initialize the Master server and the node server:
```
Comment (iphutch) [inline integration/openshift/README.md]: Use lower case for "master"

File: integration/openshift/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+```
+$ ./init.sh
+```
+
+You can also find a basic test which can be executed in order to verify that the
+environment was build correctly. To run the test you need the `bats` framework.
```
Comment (iphutch) [inline integration/openshift/README.md]: I suggest we re-write the sentence beginning on line 20:
Run a basic verification test to check the environment was built correctly (recommended).  To run the test you need the `bats` framework.

**Comment by chavafg** [comment]
@iphutch thanks for your comments, they are now fixed.

File: integration/openshift/hello_world.bats [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+@test "Hello Openshift using CC" {
+	# The below json file was taken from:
+	# https://github.com/openshift/origin/tree/master/examples/hello-openshift
+	# and modified to be an untrusted workload and then use Clear Containers.
+	sudo -E oc create -f "${BATS_TEST_DIRNAME}/data/hello-pod-cc.json"
+	wait_time=20
+	sleep_time=3
+	output_file=$(mktemp)
+	echo "$output_file"
```
Comment (devimc) [inline integration/openshift/hello_world.bats]: is this a debug message?

**Comment by devimc** [comment]
did you plan to add a teardown script for openshift ?

**Comment by devimc** [comment]
lgtm

**Pull request merged**